hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -1453,6 +1453,97 @@ def list_decommissioned_regionservers
       @admin.listDecommissionedRegionServers
     end
 
+    #----------------------------------------------------------------------------------------------
+    # Retrieve SlowLog Responses from RegionServers
+    def get_slowlog_responses(server_names, args)
+      unless server_names.is_a?(Array) || server_names.is_a?(String)
+        raise(ArgumentError,
+              "#{server_names.class} of #{server_names.inspect} is not of Array/String type")
+      end
+      if server_names == '*'
+        server_names = getServerNames([], true)
+      else
+        server_names_list = get_server_names(server_names)
+        server_names = getServerNames(server_names_list, false)
+      end
+      slow_log_responses = @admin.getSlowLogResponses(server_names)
+      filtered_log_responses = filter_slow_responses(args,
+                                                     slow_log_responses)
+      puts 'Retrieved SlowLog Responses from RegionServers.....'
```

Comment:
nit: What's with the ellipsis everywhere? Here and in log messages, they don't add anything for the user.